### PR TITLE
config-CODEOWNERS.md: draft document

### PR DESCRIPTION
**Pull request by nfrisby**
Single commit; see the new document.

### CODE REVIEW

**Comment by nfrisby** [comment]
This is my plan for the Consensus content of the CODEOWNERS file on PR #2894.

I certainly want to right this down somewhere, but I'm not 100% sure this is the right place. Objections/suggestions?

@coot We can chat more if you want to configure something similar for the Network Team files.

File: ouroboros-consensus/docs/config-CODEOWNERS.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+Because it's an IOHK Team, I can only add it to an IOHK repository. So I also
+created the temporary `input-output-hk/test-CODEOWNERS` to avoid noise in a
+genuine repository. I configured that repo as follows.
+
+  * I added a Branches rule for the `main` branch and ticked the two boxes so
+    that PRs require Approvals before merge and that in particular at least one
+    Approval must come from a code owner.
+
+  * I added Philipp, Pawel Sculz, and Jared Corduan to the repository as
```
Comment (coot) [inline ouroboros-consensus/docs/config-CODEOWNERS.md:50]:
```suggestion
  * I added Philipp, Pawel Szulc, and Jared Corduan to the repository as
```

**Comment by coot** [comment]
Pretty nice, thanks for doing due diligence :). I'll do the same for the networking team.

File: ouroboros-consensus/docs/config-CODEOWNERS.md (hunk 1 above)
Comment (nfrisby) [reply ouroboros-consensus/docs/config-CODEOWNERS.md]: Thanks very much. Fixed.

**Comment by nfrisby** [comment]
@coot Would you Approve this PR for merge?

**Review by coot — APPROVED** [review]
LGTM

**Comment by nfrisby** [comment]
I fixed a typo and rebased.

**Pull request merged**